v2/protegrity-data-tokenization/README.md

```diff
@@ -8,7 +8,7 @@ Supported data formats:
 
 - JSON
 - CSV
-- Avro
+- [Avro](#Avro-notes)
```

Comment:
Please, replace `#Avro-notes` with `#Avro-notes:` so the reference will be right 🙂

Reply:
On Github the link works correctly, but after adding ":" it doesn't work. I also checked the link in IDEA, but the link to the section doesn't work in any of these cases.

Reply:
Got you, thanks for double checking!